conf/build-ghc-app.sh

```diff
@@ -11,3 +10,8 @@ go get -d ./...
 export GOBIN='/srv/bin'
 go install ghc-app
 popd
+
+export EVENT_DIGEST_PATH="${GOPATH}/src/event-digest"
+pushd "${EVENT_DIGEST_PATH}" > /dev/null
```

Comment:
That is not true in my shell, but I do not have a reason to disbelieve you for another shell implementation.

```
/sites/fdn (jsj-private-firm-link) $ popd                                                                                   
/code/OpenEmu (master) $ pushd /sites/fdn                                                                                  
/sites/fdn (jsj-private-firm-link) $
```